sap-s4hana-soap/2.2/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -0,0 +1,558 @@
+= SAP S/4HANA SOAP Connector 2.2 Reference
+:page-aliases: connectors::sap/sap-s4hana-soap-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector for SAP S/4HANA SOAP provides full support for the SOAP APIs exposed by SAP S/4HANA ERP and from version 2.1.x provides support also for SAP S/4HANA On-Premise ERP.
+
+The main class of an extension is the entry point from which configurations, connection providers, operations, and sources are declared.
+
+== Configurations
+---
+[[Config]]
+=== Config
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<Config_BasicAuthentication, Basic Authentication>>
+* <<Config_Oauth2ClientCredentialsAuthentication, Oauth2 Client Credentials Authentication>>
+ | The connection types to provide to this configuration. | | x
+ | Encoding a| String |  Character encoding used in the messaging. |  |
+| Connection timeout a| Number |  Connection timeout for the request |  30 |
+| Connection Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use in the Timeout configurations |  `SECONDS` |
+| Encoding a| String |  Default character encoding to use in all the messages. If not specified, the default charset in the mule configuration will be used |  |
+| Name a| String |  The identifier of this element used to reference it in other components |  | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
+| Version a| Enumeration, one of:
+
+** WSA200508
+** WSA200408 |  WS-Addressing version. Defaults to WSA 2005-08. |  WSA200508 |
```

Comment:
The version in the description is different than the one in the Default column. The version in the Default column is probably the correct one.